libp2p/protocols/kademlia/lookupstate.nim

```diff
@@ -103,11 +103,6 @@ proc init*(T: type LookupState, targetId: Key, initialPeers: seq[PeerId]): T =
   )
   return res
 
-proc checkConvergence*(state: LookupState): bool =
```

Comment:
why remove this `proc` ?

Reply:
I mentioned in the text of the PR, but to expand:

this proc is only used once, in kademlia.nim. it is public, so it adds noise to the public interface, and it being in a different file, means that you have to navigate the code base to understand what's happening. On top of that, having a helper function indicates the possibility of repeated use, and the public + different file adds to this.

The purpose of helper functions is to serve the objective of making the code base more maintainable and readable. in this case, the opposite effect has taken place based on my experience working with the findNode function

instead of inlining, ive moved the helper to just below the callsite.

Reply:
> instead of inlining, ive moved the helper to just below the callsite.

:+1: